App/src/main/java/com/yagni/model/MarketRepo.java

```diff
@@ -0,0 +1,152 @@
+package com.yagni.model;
+
+import java.sql.*;
+import javax.swing.*;
+
+import com.yagni.controller.*;
+
+public class MarketRepo {
+
+    //Best customers by doller amount 
+    private static String firstCustomer;
+    private static String secondCustomer;
+    private static String thirdCustomer;
+
+    //Most ordered product
+    private static String firstProduct;
+    private static String secondProduct;
+    private static String thirdProduct;
+    private static String firstAmt;
+    private static String secAmt;
+    private static String thirdAmt;
+
+  
+     public void dayRepo(DbConnection linkDb){
```

Comment:
maybe change dayRepo method to dayReport or change weekReport, monthReport methods to weekRepo,monthRepo.  think it would look cleaner if all 3 matched. Other than that looks good, good job.

I'll approve but leave open to give others an opportunity to review/merge.

Reply:
Thanks for making this suggestion @S4kRiPp4 and pushing this change @Jesusxx-240 ! Resolving the conversation